tedana/decomposition/eigendecomp.py

```diff
@@ -167,20 +168,24 @@ def tedpca(catd, OCcatd, combmode, mask, t2s, t2sG, stabilize,
         o[mask] = eimum
         eimum = np.squeeze(o).astype(bool)
 
-        vTmix = v.T
+        vTmix = comp_ts.T
         vTmixN = ((vTmix.T - vTmix.T.mean(0)) / vTmix.T.std(0)).T
         LGR.info('Making initial component selection guess from PCA results')
-        _, ctb, betasv, v_T = model.fitmodels_direct(catd, v.T, eimum, t2s, t2sG,
-                                                     tes, combmode, ref_img,
-                                                     mmixN=vTmixN, full_sel=False)
-        ctb = ctb[ctb[:, 0].argsort(), :]
-        ctb = np.vstack([ctb.T[:3], sp]).T
+        _, ct_df, betasv, v_T = model.fitmodels_direct(catd, comp_ts.T, eimum,
+                                                       t2s, t2sG,
+                                                       tes, combmode, ref_img,
+                                                       mmixN=vTmixN,
+                                                       full_sel=False)
+        ct_df['variance explained 2 (normalized)'] = varex_norm
 
         # Save state
         fname = op.abspath('pcastate.pkl')
         LGR.info('Saving PCA results to: {}'.format(fname))
-        pcastate = {'u': u, 's': s, 'v': v, 'ctb': ctb,
-                    'eigelb': eigelb, 'spmin': spmin, 'spcum': spcum}
+        pcastate = {'u': voxel_comp_weights, 'varex': varex, 'comp_ts': comp_ts,
```

Comment:
Why are we keeping the `u` here ?

Reply:
I'll replace `u` with `voxel_comp_weights` (or something better if you can think of it).